### PR TITLE
[gohai][processes] Simplify payload

### PR DESCRIPTION
**Pull request by olivielpeau**
- Only send a processes payload when gohai actually returns data
- Remove the `format_description` and `format_version` fields

### CODE REVIEW

File: checks/collector.py [hunk 1]
```diff
@@ -383,19 +369,18 @@ def run(self, checksd=None, start_event=True, configs_reloaded=False):
...
+                    processes_snaps = gohai_processes_json.get('processes')
+                    if processes_snaps:
+                        processes_payload = {
+                            'snaps': [processes_snaps]
+                        }
+
+                        payload['resources'] = {
+                            'processes': processes_payload,
+                            'meta': {
+                                'host': payload['internalHostname'],
```
Comment (degemer) [inline checks/collector.py]: Why not use `self.hostname` ? I think it carries more sense than `paylaod['internalHostname]` (while being the same thing).

**Comment by degemer** [comment]
LGTM apart from the small remark.
I trust you on the uselessness of `format_description` and `format_version`, I didn't check.

File: checks/collector.py (hunk 1 above)
Comment (olivielpeau) [inline checks/collector.py]: For no reason other than my own laziness to change it ;) Thanks for spotting this!

**Comment by degemer** [comment]
👍

**Pull request merged**